volume/volume.go

```diff
@@ -108,6 +120,9 @@ func (vdm *VolumeDriverManager) Path(volumeName, volumeID string) (string, error
 
 func (vdm *VolumeDriverManager) Create(volumeName string, volumeOpts VolumeOpts) error {
 	for _, driver := range vdm.Drivers {
+		log.WithFields(log.Fields{
+			"volumeName": volumeName,
+			"volumeOpts": golf.Fore("volumeOpts", volumeOpts)}).Debugf("%s: Create()", driver.Name())
```

Comment:
Hi @clintonskitson,

This is not necessary. The `logrus` branch we use automatically unwinds fields using `Golf`. You don't need to reference `Golf` at all anywhere in this file, and the package import should be removed.